### PR TITLE
Add SimbaChain swag

### LINKED ISSUE #185: Add 7PHP swag
## Wild swag opportunity appeared!

- [x] I've checked that this isn't a duplicate issue.

Q. Which company offers it?  
A. : [7PHP](https://7php.com/)

Q. Where have you seen the announcement (URL)?  
A. : https://7php.com/stickers/

Q. What kind of swag (clothing, stickers, ...)?  
A. : stickers

Q. Have you received swag from this?  
A. : No

Q. How long did it take?  
A. : N/A

Q. Does it ship worldwide?  
A. : Yes?

Note I don't think we should add it to the list as it's not really interesting, but it's still free swag..

- [x] I've checked that the promotion is legit.
- [x] I've checked that it's still available to the public.

### LINKED ISSUE #196: Add SimbaChain swag
## Wild swag opportunity appeared!

- [x] I've checked that this isn't a duplicate issue.

Q. Which company offers it?  
A. : SimbaChain

Q. Where have you seen the announcement (URL)?  
A. : https://simbachain.com/community/join-our-community/

Q. What kind of swag (clothing, stickers, ...)?  
A. : clothing (t-shirt)

Q. Have you received swag from this?  
A. : No

Q. How long did it take?  
A. : N/A

Q. Does it ship worldwide?  
A. : Yes

Comes from this discussion: https://github.com/ethereum/go-ethereum/issues/2104#issuecomment-399721639

- [x] I've checked that the promotion is legit.
- [x] I've checked that it's still available to the public.

### PR DESCRIPTION
**Pull request by vivekptnk**
## Wild swag opportunity appeared

- [x] I've checked that this isn't a duplicate pull request.
- [x] I've checked that this is linked to a new swag opportunity issue <!>

Implement swag opportunity: SimbaChain

Closes #196

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -206,6 +206,15 @@
...
   },
+  {
+    "name": "SimbaChain",
+    "difficulty": "easy",
+    "description": "Register and get a free Blockchain Shirt! Pick your side, 'eth_listtransactions = _yep or eth_listtransactions = _nope'.",
+    "reference": " https://simbachain.com/community/join-our-community/",
+    "image": "https://simbachain.com/wp-content/uploads/2018/07/SIMBA-Blockchain-Shirt-V2.png",
+    "dateAdded": "2018-12-04T5:58:05.000Z",
+    "tags": ["clothing"]
+  }
```
Comment (aslafy-z) [inline data.json]: It misses a ','

```suggestion
  },
```

File: data.json [hunk 2]
```diff
@@ -206,6 +206,15 @@
     "dateAdded": "2018-10-12T13:10:59.000Z",
     "tags": ["clothing", "hacktoberfest"]
   },
+  {
+    "name": "SimbaChain",
+    "difficulty": "easy",
+    "description": "Register and get a free Blockchain Shirt! Pick your side, 'eth_listtransactions = _yep or eth_listtransactions = _nope'.",
```
Comment (aslafy-z) [inline data.json]:
```suggestion
    "description": "Register and get a free Blockchain Shirt! Pick your side, 'eth_listtransactions = _yep' vs 'eth_listtransactions = _nope'.",
```

File: data.json [hunk 3]
```diff
@@ -206,6 +206,15 @@
     "dateAdded": "2018-10-12T13:10:59.000Z",
     "tags": ["clothing", "hacktoberfest"]
   },
+  {
+    "name": "SimbaChain",
+    "difficulty": "easy",
+    "description": "Register and get a free Blockchain Shirt! Pick your side, 'eth_listtransactions = _yep' vs 'eth_listtransactions = _nope'.",
```
[the remark this answers is not in the document]
Comment (aslafy-z) [reply data.json]: @vivekptnk you can wrap each single quoted block with `<code></code>` blocks

File: data.json [hunk 4]
```diff
@@ -209,7 +209,7 @@
   {
     "name": "SimbaChain",
     "difficulty": "easy",
-    "description": "Register and get a free Blockchain Shirt! Pick your side, 'eth_listtransactions=_yep' vs 'eth_listtransactions=_nope'.",
+    "description": "Register and get a free Blockchain Shirt! Pick your side, <code>'eth_listtransactions = _yep'</code> vs <code>'eth_listtransactions=_nope'</code>.",
```
Comment (aslafy-z) [inline data.json]: You want to delete the quotes and add back space between 'transactions = _nope'

**Comment by aslafy-z** [comment]
Still need to add some styles for `<code>` tags

File: data.json (hunk 4 above)
Comment (vivekptnk) [reply data.json]: what kind of styles do you think, i was thinking font-size and maybe color

**Comment by vivekptnk** [comment]
> Still need to add some styles for `<code>` tags

What kind of styles? Font Size, color?

**Comment by aslafy-z** [comment]
I was thinking about something like this:



It has to go to another PR anyway

**Comment by aslafy-z** [comment]
@vivekptnk please delete your last commit as we will style the code tag project wide.

**Comment by vivekptnk** [comment]
> @vivekptnk please delete your last commit as we will style the code tag project wide.

gotcha, reverted the commit

**Comment by aslafy-z** [comment]
#239 has been merged

**Comment by aslafy-z** [comment]
Please rebase this PR so we can merge it. @vivekptnk

**Comment by vivekptnk** [comment]
> Please rebase this PR so we can merge it. @vivekptnk

Done 👍. Please check.

**Pull request merged**